src/pages/letter/LetterReplyPage.tsx

```
@@ -1,47 +1,87 @@
+import { useLetterDetail } from '@/hooks/letters/useLetterDetail';
 import { useMemo } from 'react';
-import { useNavigate } from 'react-router-dom';
 
 import BackHeader from '@/components/common/headers/BackHeader';
 import { Button } from '@/components/common/Button';
+import { LoadingDots } from '@/components/LoadingDots';
+import NotFoundPage from '../system/NotFoundPage';
+import LetterCard from '@/components/letters/LetterCard';
+import { DEFAULT_FONT_ID, FONT_ASSET_MAP } from '@/constants/fontAssets';
+import { DEFAULT_PAPER_ID, PAPER_ASSET_MAP } from '@/constants/paperAssets';
+import { useNavigate } from 'react-router-dom';
```

Biome (2.3.13)

[error] Shouldn't redeclare 'useNavigate'. Consider to delete it or rename it.

'useNavigate' is defined here:


(lint/suspicious/noRedeclare)